blocks/card/card.css

```diff
@@ -0,0 +1,145 @@
+.card{
+    padding:20px 10px 60px;
+}
+.card__items{
+    list-style: none;
+    padding: 0;
+}
+.card__card{
+    padding: 20px 0;
```

Comment:
Seems padding sits outside the card and doubles the space between each card, which doesn't seem like it was the intended purpose? This seems like a good place for `padding:0` then `margin-bottom: ##px;` then `card__card:last-child {margin-bottom: 0;}`